epistemix_platform/infrastructure/tests/ecr/test_simulation_runner_repository_template.py

```
@@ -572,3 +570,135 @@ def test_ecr_scan_event_rule_not_present(self, ecr_template: dict[str, Any]):
+        Requires: cfn-guard binary (pre-built from AWS)
+        Install: ./scripts/install-cfn-guard.sh
+        Rules: guard_rules/ecr/ecr_security_rules.guard
+        Docs: guard_rules/README.md
+        """
+        import subprocess
+
+        rules_path = (
+            Path(__file__).parent.parent.parent / "guard_rules" / "ecr" / "ecr_security_rules.guard"
+        )
+
+        result = subprocess.run(
```

Ruff (0.14.2)

`subprocess` call: check for execution of untrusted input

(S603)